gems/aws-sdk-finspace/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Issue - Add a `Content-Type` header to mitigate a service side issue.
```

Comment:
Let's note what the impacted services were.

Reply:
Each service has their own change log..